package.json

```diff
@@ -58,7 +62,6 @@
     "url": "git+https://github.com/duneanalytics/hooks.git"
   },
   "dependencies": {
-    "react-dom": "^18.3.1",
```

Comment:
This should never be a dependency as it is installed on consumer side with v18, althought the consumer could be on a different major version. 

We need to have it in `devDependencies` to be used by this lib itself and theoretically in `peerDependencies` to declare compatibility.